app.js

```diff
@@ -38,14 +41,14 @@ export const MENU = [
 export const images = [
   {
     index:
-      'https://static01.nyt.com/images/2023/12/12/climate/12cli-cats/12cli-cats-jumbo.jpg?quality=75&auto=webp',
+      "https://static01.nyt.com/images/2023/12/12/climate/12cli-cats/12cli-cats-jumbo.jpg?quality=75&auto=webp",
```

Comment:
These changes don't mean anything, they just change `'` to `"`. This is because your group does not have consistent formatting settings, and results in lots of little changes in PRs that don't have any meaningful impact, but cost extra effort for reviewers to sift through, which can mean that reviewers miss more important changes. I recommend using something like [Prettier](https://prettier.io) in future projects.